File: 프로그래머스/0/181836. 그림 확대/그림 확대.cpp
#include <string>
#include <vector>

using namespace std;

vector<string> solution(vector<string> picture, int k) {
    vector<string> answer(picture.size() * k);
    for(int i=0;i<answer.size();i++){
        string tmp= "";
        for(int j=0;j<picture[i/k].length(); j++){
            for(int l=0;l<k;l++)
                tmp += picture[i/k][j];
            
        }
        answer[i] = tmp;
    }
    return answer;
}

File: 프로그래머스/0/181836. 그림 확대/그림 확대.java
class Solution {
    public String[] solution(String[] picture, int k) {
        String[] answer = new String[picture.length * k];
        for(int i=0;i<answer.length;i++){
            StringBuilder sb = new StringBuilder();
            for(int j=0;j<picture[i/k].length(); j++){
                for(int l=0;l<k;l++){
                    sb.append(picture[i/k].charAt(j));
                }

            }
            answer[i] = sb.toString();
        }

        return answer;
    }
}